### PR TITLE
Add utility to find model labels

### PR DESCRIPTION
**Pull request by sgugger**
# What does this PR do?

This PR adds a utility function to find the labels of a model. It will be useful for Keras and the Trainer (integration with the first is left for a follow-up PR).

### CODE REVIEW

File: src/transformers/utils/generic.py [hunk 1]
```diff
@@ -289,3 +290,23 @@ def __enter__(self):
...
+    """
+    model_name = model_class.__name__
+    if model_name.startswith("TF"):
+        signature = inspect.signature(model_class.call)
+    elif model_name.startswith("Flax"):
+        signature = inspect.signature(model_class.__call__)
+    else:
+        signature = inspect.signature(model_class.forward)
+    if "QuestionAnswering" in model_name:
+        return [p for p in signature.parameters if "label" in p or p == "start_positions" or p == "end_positions"]
```
Comment (Rocketknight1) [inline src/transformers/utils/generic.py:310]:
```suggestion
        return [p for p in signature.parameters if "label" in p or p in ("start_positions", "end_positions")]
```
This is the nittiest nit ever but it's a little cleaner!

**Review by Rocketknight1 — APPROVED** [review]

File: src/transformers/utils/generic.py (hunk 1 above)
Comment (sgugger) [reply src/transformers/utils/generic.py:310]: Indeed!

**Review by LysandreJik — APPROVED** [review]
Very nice, LGTM!

**Comment by LysandreJik** [comment]
cc @patrickvonplaten for knowledge

**Review by patrickvonplaten — COMMENTED** [review]
Very nice addition

**Pull request merged**